addons/gamification/data/goal_base.xml

```diff
@@ -20,76 +20,232 @@
+                                        %if line['suffix'] or line['monetary']:
+                                            (${line['full_suffix']})
+                                        %endif
+                                    </td>
+                                    <td style="padding: 20px 0;" align="right">${"%.2f" %line['target']}
+                                    % if line['suffix']:
+                                        ${line['suffix']}
+                                    % endif
+                                    </td>
+                                    <td style="padding: 20px 0;" align="right">${"%.2f" %line['current']}
+                                    % if line['suffix']:
+                                        ${line['suffix']}
+                                    % endif
+                                    </td>
+                                    <td style="padding: 20px 0;font-size:25px;color:#9A6C8E;" align="right"><strong>${line['completeness']| int}%</strong></td>
+                                </tr>
+                                <tr>
+                                    <td colspan="5" style="height:1px;background-color:#e3e3e3;"></td>
+                                </tr>
+                            % endfor        
+                            </tbody>
+                        </table>                   
+                    % else: 
+                        <span style="color:#A8A8A8;font-size:13px;">
+                            The following message contains the current progress for the challenge <strong>${object.name}</strong>.
+                        </span> 
+                        % for line in context:
+                            <!-- Header + Button table -->
+                            <table cellspacing="0" cellpadding="0" width="100%"style="margin-top:35px;">
+                                <tr><td width="50%">
+                                    <p style="font-size:22px;color:#666666;">Top Achievers for goal <strong>${line['name']}</strong></p>
+                                    </td>
+                                </tr>
+                            </table>
+                            <!-- Podium -->
+                            % if len(line['goals'])>2:
+                                <table cellspacing="0" cellpadding="0" width="100%" style="margin-top:10px;">
+                                    <tr><td style="padding:0 30px;">
+                                        <table cellspacing="0" cellpadding="0" width="100%">
+                                            <tr>
+                                                % set top_goals = [line['goals'][1], line['goals'][0], line['goals'][2]]
+                                                % for goal in top_goals:
+                                                    <td align="center" style="width:32%;"
+                                                        % if loop.index == 1:
```

Comment:
I think it does not change a thing whether you define it inside or outside the tag. I followed the old template example.